lib/Dialect/TritonGPU/Transforms/AccelerateAMDMatmul.cpp

```diff
@@ -63,13 +61,23 @@ warpsPerTileMFMA(tt::DotOp dotOp, const ArrayRef<int64_t> shape, int numWarps) {
   return ret;
 }
 
+SmallVector<unsigned, 2>
+warpsPerTileMFMA(tt::DotOp dotOp, const ArrayRef<int64_t> shape, int numWarps) {
+  return warpsPerTile(dotOp, shape, numWarps, {32, 32});
+}
+
+SmallVector<unsigned, 2>
+warpsPerTileWMMA(tt::DotOp dotOp, const ArrayRef<int64_t> shape, int numWarps) {
+  return warpsPerTile(dotOp, shape, numWarps, {16, 16});
+}
+
 class BlockedToMFMA : public mlir::RewritePattern {
   int mfmaVersion;
   int enforcedNonKDim;
 
 public:
   BlockedToMFMA(mlir::MLIRContext *context, int mfmaVersion, int nonKDim)
-      : mlir::RewritePattern(tt::DotOp::getOperationName(), 2, context),
+      : mlir::RewritePattern(tt::DotOp::getOperationName(), 3, context),
```

Comment:
because another pattern was introduced. In my understanding it is better to have different priorities. And MFMA looks better to use (but impossible to use them on a single device for now of course).

Reply:
MFMA and WMMA are mutually exclusive, so I think you can use same priority